chapter_assets/slack-templates/Out-of-Subject-Pots-in-#barcelona-chapter.md

```diff
@@ -0,0 +1,44 @@
+Out of subjects posts in #barcelona-chapter: templates of messages
+---
+
+# Context
+This template should be used When someone/some organization posts something that is not following the channel description/guidelines:
+"codebar Barcelona news or anything related to promote diversity in tech in Barcelona"
+
+# 1st quick personal message to poster
+I’m {{your name}} one of the organizers of the codebar :codebar: Barcelona chapter.
+I have just seen your post in #barcelona-chapter. How does this event
+promote diversity in tech in Barcelona?
+If it doesn’t, please take it down from this channel that is limited to codebar Barcelona chapter news or events that promote diversity in tech in Barcelona.
+You are welcome to publish it in #barcelona-social.
+If you have any questions or comments do not hesitate to get in touch with me.
+
+# More formal message strips if the first one is not sufficient
+ 
+## unclear rule?
+
+The rule is clear, if the event promotes diversity in tech, it can be in
+the #barcelona-chapter. If not it might be welcome in #barcelona-social
+
+## event without diversity or code of conduct
+
+I had a look at your event. I didn’t see any talks related to diversity,
+and I haven’t see a code of conduct that assure it to be a safe place.
+
+## code of conduct
+Regarding the code of conduct, it is one of the best way to feel people safe. In my opinion  the CNCF one is really great (https://github.com/cncf/foundation/blob/master/code-of-conduct.md) but some are simpler likes codebar’s code of conduct (https://codebar.io/code-of-conduct)
+
+## we are defending our positions
+
+The #barcelona-chapter channel is only for announcing codebar events or other events in Barcelona that promotes diversity in tech. This is not a public place where anybody can announce anything. We actively take care of it by checking the content, the diversity of the speakers, the existence of a code of conduct and if it is free or diversity tickets exist.  In the past we have asked announcers to take down their message or taken them down ourself when needed and will continue to do so.
+
+## diversity is not 'everybody is welcome'
+
+We do not believe that diversity can be achieved by just saying ‘everybody is welcome’. It needs to be unambiguously encouraged and members of underrepresented group need to fell safe and protected. That is the first role of the code of conduct. The second one is to announce to all the (other) attendees that diversity is encouraged ( and protected). We are open to talk about it in detail if you want to know more about diversity and how we see it and promote it. All codebar activities are made under codebar’s code of conduct. Maybe we aren’t
+clear enough with the host about that fact.
```

Comment:
This suggests the question: Are we being clear enough about this when our hosts? In the past we have asked coaches that come from the host company to join codebar and read the code of conduct. Do we need to be more explicit?

Reply:
imo, yes

Reply:
should we start another initiative to update our doc/process?